conformance/utils/kubernetes/helpers_test.go

```diff
@@ -0,0 +1,160 @@
+package kubernetes
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/gateway-api/apis/v1alpha2"
+	"sigs.k8s.io/gateway-api/apis/v1beta1"
+)
+
+func TestSupportedKindsMatch(t *testing.T) {
+	tests := []struct {
+		name     string
+		expected []v1beta1.ListenerStatus
+		actual   []v1beta1.ListenerStatus
+		want     bool
+	}{
+		{
+			name: "expected empty and actual is non empty",
+			expected: []v1beta1.ListenerStatus{
+				{
+					Name:           v1beta1.SectionName("https"),
+					SupportedKinds: []v1beta1.RouteGroupKind{},
+					Conditions: []metav1.Condition{{
+						Type:   string(v1beta1.ListenerConditionResolvedRefs),
+						Status: metav1.ConditionFalse,
+						Reason: string(v1beta1.ListenerReasonRefNotPermitted),
+					}},
```

Comment:
Do these ever actually vary? Can we either omit them entirely or use something like a `commonConditions` var?

Reply:
sure will rm